src/test/java/com/wikia/webdriver/common/contentpatterns/AdsFandomContent.java

```diff
@@ -26,8 +26,9 @@
     SLOT_SELECTORS.put(TOP_BOXAD, "div[id$='top_boxad_0__container__']");
     SLOT_SELECTORS.put(INCONTENT_BOXAD, "div[id$='incontent_boxad_0__container__']");
     SLOT_SELECTORS.put(BOTTOM_BOXAD, "div[id$='bottom_boxad_0__container__']");
-    SLOT_SELECTORS.put(FEED_BOXAD, "div[id$='feed_boxad_0__container__']");
+    SLOT_SELECTORS.put(FEED_BOXAD, "div[id$='google_ads_iframe_/5441/wka1a.PF/feed_boxad/desktop/ns-article/_fandom-all_0__container__']");
 
+    // TODO move to ad slots
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1135)